org-eclipse-che-jdt-ls-extension/jdt.ls.extension.core/src/main/java/org/eclipse/che/jdt/ls/extension/core/internal/ChangeUtil.java

```diff
@@ -88,6 +91,31 @@ public static void convertChanges(Change change, CheWorkspaceEdit edit) throws C
     }
   }
 
+  /**
+   * Converts {@link RefactoringStatus} to dto object {@link
+   * org.eclipse.che.jdt.ls.extension.api.dto.RefactoringStatus}.
+   *
+   * @param status the object to be converted
+   * @return dto object which describes status of the refactoring
+   */
+  public static org.eclipse.che.jdt.ls.extension.api.dto.RefactoringStatus convertRefactoringStatus(
```

Comment:
Can rename  `RefactoringStatus` to get rid of fqn usage in code?

Reply:
I think it's more important to have good API than to save code in an implementation class. I would vote for keeping the nice name in the API (unless you have an idea for the new name?)